framework/auth/views.py

```diff
@@ -150,7 +154,7 @@ def auth_login(auth, **kwargs):
 
     status_message = request.args.get('status', '')
     if status_message == 'expired':
-        status.push_status_message('The private link you used is expired.')
+        status.push_status_message('The private link you used is expired.  Please <a href="/settings/account/resend email.')
```

Comment:
This is not valid HTML.

Also, you should pass the `trust=True` param to make it clear that this message is safe to render as HTML as-is.